src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in [stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl in
+ * [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styeObject = {};
+  const stylesArray = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  stylesArray.forEach((style) => {
+    const [key, value] = style.split(':');
+
+    styeObject[key.trim()] = value.trim();
```

Comment:
You use the trim() function 3 times, you can move this functionality into a separate function

Reply:
Can't agree with this one
Why you need to write a separate function if you can use one method instead